metadata.json

```diff
@@ -1,14 +1,94 @@
 {
   "name": "puppetlabs-nodejs",
-  "version": "0.7.0",
+  "version": "0.8.0",
   "author": "puppetlabs",
-  "summary": "Install nodejs package and npm package provider for Debian, Ubuntu, Fedora, RedHat, Gentoo, and Archlinux.",
+  "summary": "Install Node.js package and npm package provider.",
   "license": "Apache-2.0",
   "source": "https://github.com/puppetlabs/puppetlabs-nodejs",
   "project_page": "https://github.com/puppetlabs/puppetlabs-nodejs",
   "issues_url": "https://tickets.puppetlabs.com/browse/MODULES",
+  "operatingsystem_support": [
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "OracleLinux",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "5",
+        "6",
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "7"
+      ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "10.04",
+        "12.04",
+        "14.04"
+      ]
+    }
+  ],
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">=2.0.0"},
-    {"name":"puppetlabs/apt","version_requirement":">=0.0.3"}
+    {
+      "name": "puppetlabs/apt",
+      "version_requirement": ">=1.5.0 <2.0.0"
+    },
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">=2.2.1 <5.0.0"
+    },
+    {
+      "name": "gentoo/portage",
+      "version_requirement": ">=2.0.1 <3.0.0"
+    },
+    {
+      "name": "rismoney/chocolatey",
+      "version_requirement": ">=0.0.3 <1.0.0"
+    },
+    {
+      "name": "stahnma/epel",
+      "version_requirement": ">=1.0.0 <2.0.0"
+    },
+    {
+      "name": "treydock/gpg_key",
+      "version_requirement": ">=0.0.3 <1.0.0"
+    }
```

Comment:
All of these platform-dependent requirements shouldn't be in the metadata, but rather documented in the README.md similar to https://github.com/puppetlabs/puppetlabs-rabbitmq#module-dependencies